server/handler.go

```diff
@@ -182,6 +184,22 @@ func (h *Handler) handleValidity(w http.ResponseWriter, req *http.Request) {
 	replyOK(w, emptyMapCBOR, mimeTypeValidity)
 }
 
+func (h *Handler) handleHealth(w http.ResponseWriter, req *http.Request) {
+	ac := h.CertManager.GetAugmentedChain()
+	if ac == nil {
+		replyError(w, http.StatusNotFound)
+		return
+	}
+	err := ac.VerifyAll(timeutil.Now(), false)
```

Comment:
Would it make sense to change `false` to `!AllowTestCert`? https://github.com/google/webpackager/blob/1026e99cbadb70c6967fc2937e50260dbfb83934/cmd/webpkgserver/webpkgserver.example.toml#L200-L209

Reply:
Yes, updated.